tests/System.Numerics.Experimental.Tests/HalfTests.cs

```diff
@@ -0,0 +1,364 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace System.Numerics.Experimental.Tests
+{
+    public partial class HalfTests
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why add these for tests?

Reply:
I just copied the existing method template from BitConverter and didnt consider removing it.